shared/chat/conversation/messages/wrapper/long-pressable/index.native.tsx

```diff
@@ -25,7 +25,6 @@ const LongPressable = (props: {children: React.ElementType; onSwipeLeft: () => v
       onSwipeableRightWillOpen={onRightOpen}
       friction={2}
       rightThreshold={100}
-      failOffsetX={0}
```

Comment:
doesn't actually exist as a prop : cc: @mmaxim

Reply:
We can't remove this, it exists on the underlying thing. Removing this will break swiping back to the inbox, just ts-ignore.